assets/js/modules/optimize/util/validation.js

```diff
@@ -25,7 +25,7 @@
  * @return {boolean} True if the given optimize ID is valid, false otherwise.
  */
 export function isValidOptimizeID( optimizeID ) {
-	return typeof optimizeID === 'string' && !! optimizeID.match( /^OPT-[A-Z0-9]+$/ );
+	return typeof optimizeID === 'string' && !! optimizeID.match( /^(GTM|OPT)-[a-zA-Z\d]{7}$/ );
```

Comment:
Let's not require 7 characters specifically as that makes this match fragile to change.

Let's keep the previous pattern (with the addition of the possible `GTM` prefix). I'm not sure if the ID is case-sensitive, but if it isn't, we can make the whole match case-insensitive by adding an `i` after the slash at the end as a flag. I think this is a good idea, but let's find out for sure first.

Reply:
Done